src/pages/task/metadata/index.tsx

```diff
@@ -230,6 +235,18 @@ export const Metadata: React.VFC<Props> = ({
           </StyledLink>
         </MetadataItem>
       )}
+      {podLink && (
+        <MetadataItem>
+          Container:{" "}
+          <StyledLink
+            data-cy="task-pod-link"
+            href={podLink}
```

Comment:
nit. I think you could avoid storing the `hostLink`/`podLink` variables by doing something along the lines of:
```suggestion
      {isContainerTask && (
        <MetadataItem>
          Container:{" "}
          <StyledLink
            data-cy="task-pod-link"
            href={getPodRoute(podId)}
```